Taskfile.yaml

```diff
@@ -176,7 +176,7 @@ tasks:
       - featurebyte/**/*
     cmds:
       - poetry run pip-licenses --packages '{{ .PACKAGES }}' --allow-only='{{ .PERMISSIVE_LICENSES }}'
-      - poetry run pip-audit --ignore-vuln PYSEC-2023-177
+      - poetry run pip-audit --ignore-vuln PYSEC-2023-177 --ignore-vuln GHSA-g4mx-q9vg-27p4
```

Comment:
Shouldn't be needed once https://github.com/featurebyte/featurebyte/pull/1830 is merged.

Reply:
Will update this.

Reply:
Will remove the ignore vuln in later PR (likely in https://github.com/featurebyte/featurebyte/pull/1831) once https://github.com/featurebyte/featurebyte/pull/1830 is merged.